From e14a112da27c9f8ffb2fa4757056f21052a9cb22 Mon Sep 17 00:00:00 2001
From: svenate <salini.venate@intel.com>
Date: Mon, 14 Dec 2020 10:39:52 +0530
Subject: [PATCH] Update Light HAL service name

Update Light HAL service name in file_context

Tracked-On: None
Signed-off-by: svenate <salini.venate@intel.com>
---
 vendor/file_contexts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git vendor/file_contexts vendor/file_contexts
index 1b2bc2357..5bc817ec7 100644
--- vendor/file_contexts
+++ vendor/file_contexts
@@ -45,7 +45,7 @@
 /(vendor|system/vendor)/bin/hw/android\.hardware\.keymaster@4\.1-service      u:object_r:hal_keymaster_default_exec:s0
 /(vendor|system/vendor)/bin/hw/android\.hardware\.light@2\.0-service          u:object_r:hal_light_default_exec:s0
 /(vendor|system/vendor)/bin/hw/android\.hardware\.light@2\.0-service-lazy     u:object_r:hal_light_default_exec:s0
-/(vendor|system/vendor)/bin/hw/android\.hardware\.lights-service\.example     u:object_r:hal_light_default_exec:s0
+/(vendor|system/vendor)/bin/hw/android\.hardware\.lights-service\.intel       u:object_r:hal_light_default_exec:s0
 /(vendor|system/vendor)/bin/hw/android\.hardware\.lowpan@1\.0-service         u:object_r:hal_lowpan_default_exec:s0
 /(vendor|system/vendor)/bin/hw/android\.hardware\.memtrack@1\.0-service       u:object_r:hal_memtrack_default_exec:s0
 /(vendor|system/vendor)/bin/hw/android\.hardware\.nfc@1\.0-service            u:object_r:hal_nfc_default_exec:s0
-- 
2.17.1

